frontend/src/containers/App.jsx

```diff
@@ -22,6 +23,15 @@ export default function App() {
         <AuthenticationContextProvider>
           <BrowserRouter>
             <Switch>
+              <ProtectedRoute
+                exact
+                path="/play/invalid-role"
+                component={InvalidRolePage}
+              >
+                <PlayingScenarioContextProvider>
```

Comment:
This has been replaced - and even for current PR, it could be argued that this is dead code.

Let's remove it :)

Reply:
Removed the playingscenariocontext provider from code